# EthernetPacket Message

std_msgs/Header header

std_msgs/Byte[] pkt
std_msgs/Float32 ts
